ja/faq/netlify-deployment.md

```diff
@@ -23,10 +23,25 @@ Netlify ダッシュボードの _"New site from Git"_ ボタンを押下しま
 
 ## 操作：
 
+### 静的に生成されたサイトの場合
+
 1. __ブランチをデプロイする：__ `master` もしくはデプロイしたいブランチ
 2. __ビルドコマンド：__ `npm run generate`
 3. __公開ディレクトリ：__ `dist`
 
+
+## SPAモードで生成されたサイトの場合
+
+1. __ブランチをデプロイする:__ `master` もしくはデプロイしたいブランチ
+2. __ビルドコマンド:__ `npm run build`
+3. __公開ディレクトリ:__ `dist`
+
+
+シングルページアプリケーションの場合、Netlify は再読み込み時にデフォルトで *"404 not found"* にリダイレクトしますが、[リダイレクト設定](https://www.netlify.com/docs/redirects/#rewrites)を行う事で防げます。 また、シングルページアプリケーションのヘッダーとリダイレクト両方の設定をサポートする[nuxt-netlify](https://www.bazzite.com/docs/nuxt-netlify)モジュールを利用する事もできます。
```

Comment:
```suggestion
シングルページアプリケーションの場合、Netlify は再読み込み時にデフォルトで *"404 not found"* にリダイレクトしますが、[リダイレクト設定](https://www.netlify.com/docs/redirects/#rewrites-and-proxying)を行う事で防げます。 また、シングルページアプリケーションのヘッダーとリダイレクト両方の設定をサポートする [nuxt-netlify](https://www.bazzite.com/docs/nuxt-netlify) モジュールを利用する事もできます。
```

+ Please put a space between half-width characters and full-width characters.
+ Fix the URL `https://www.netlify.com/docs/redirects/#rewrites` to `https://www.netlify.com/docs/redirects/#rewrites-and-proxying`